app/routes/task_routes.py

```diff
@@ -0,0 +1,100 @@
+from flask import Blueprint,request,jsonify,abort, make_response
+import requests, json
+from app import db
+from app.models.task import Task
+from sqlalchemy import asc
+from sqlalchemy import desc
+from app.routes.routes_helper import get_one_obj_or_abort
+from app.routes.routes_helper import validate_id
+from datetime import date
+
+task_bp = Blueprint("task_bp", __name__, url_prefix ="/tasks")
+
+def post_message_to_slack(text):
+    return requests.post('https://slack.com/api/chat.postMessage', {
+        'channel': 'task-notifications',
+        'text': text}).json()	
+
+def get_one_task_or_abort(task_id):
+    matching_task = get_one_obj_or_abort(Task, task_id)
+    if not matching_task:
+        response_str = f"Task with id {task_id} not found in database"
+        abort(make_response(jsonify({"message": response_str}),404))
+    return matching_task
+
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json() 
+    if "title" not in request_body or \
+        "description" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
```

Comment:
It's good that you're doing data validation, but you should also indicate to the user what data was invalid.